lib/assessment.js

```diff
@@ -20,9 +38,17 @@ var sum = function sum() {
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  //create a new empty array
+  //loop through length and push default Value to the new empty arrays
+    var Array = [];
```

Comment:
As a rule, don't capitalize any variables in JavaScript unless they're holding/referring to Constructor functions. 

However, this will still work as written.